zeta/README.md

```diff
@@ -0,0 +1,283 @@
+
+## Path & Landfall
+
+Hurricane Zeta started off as a Category 1 Hurricane when it hit the Yucatan Peninsula, weakened to a tropical storm, but then quickly evolved into a Category 3 Hurricane that made landfall in southeastern Louisiana. Hurricane Zeta’s fast forward motion brough strong winds inland into certain parts of the southeastern United States. The hurricane resulted in 5 fatalities and $4.4 billion in damage in the United States.
+
+<img src="images/path.png" width="650">
+
+Image Source: https://weather.com/storms/hurricane/news/2020-10-29-hurricane-zeta-tropical-storm-forecast-south-east
+
+## Storm Surge
+
+Storm surge is defined as the abnormal rise of water generated by a storm, above the predicted astronomical tide, and is expressed in terms of height above normal tide levels. Hurricane Zeta produced storm surge inundation of 6-10 feet above ground level (AGL) along the Mississippi Coast and along the Alabama coast west of Mobile Bay. The highest observation in Louisiana was recorded to be 7.0 ft, north of Grand Isle, but the storm surge hindcast suggests that this was a result of Zeta’s winds piling water up against the west side. The severe “storm surge flooding caused hundreds of roads to become impassable in the Bay St. Louis area, and portions of the U.S. Highway 90 were underwater in Harrison County. According to the NOAA report, “generally minor damage was reported in other areas of southern Mississippi.”
+
+## Rainfall & Flooding
+
+Hurricane Zeta resulted in flooded streets in the Mexican state of Quintana Roo, but there was no significant damage from the flooding. In the United States, 4-6 inches of rain was observed near Zeta’s landfall location in southeastern Louisiana through southeastern Mississippi and western Alabama. However, something to note is that the fast speed of Zeta limited the rainfall and flooding impacts in the areas that it hit. According to NOAA, “an enormous amount of rainfall reports were received from Zeta in both its tropical and extratropical stages.” Furthermore “the precursor disturbance of Zeta led to heavy rain and flooding in Jamaica, although no rainfall totals are available.”
+
+From the NOAA Hurricane Report:
+
+> Freshwater flooding across southern Alabama and the western Florida Panhandle was exacerbated by storm surge, which prevented rainwater from draining into adjacent tidal rivers, bays, or the Gulf of Mexico. Major-to-moderate flooding occurred along many rivers, with several river gauges measuring their highest level on record.
+
+## Topography & Bathymetry Data
+
+Topography data can be obtained from https://download.gebco.net/. 
+
+The topography file used for this example covers the area between 42 degrees to the north, 12 degrees to the south, -100 degrees to the west and -70 degrees to the east. An ASCII, Grid/TID Grid file was used.
```

Comment:
You should probably mention the resolution of the data that you used as that can change over time.